File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java

class Solution {
    public int[] topKFrequent(int[] nums, int k) {

        int n = nums.length;
        int[] freq = new int[n];

        for (int i = 0; i < n; i++) {
            int count = 0;
            for (int j = 0; j < n; j++) {
                if (nums[i] == nums[j]) {
                    count++;
                }
            }
            freq[i] = count;
        }

        Integer[] index = new Integer[n];
        for (int i = 0; i < n; i++) {
            index[i] = i;
        }

        Arrays.sort(index, (a, b) -> freq[b] - freq[a]);

        LinkedHashSet<Integer> set = new LinkedHashSet<>();

        for (int i = 0; i < n && set.size() < k; i++) {
            set.add(nums[index[i]]);
        }

        int[] ans = new int[set.size()];
        int i = 0;
        for (int num : set) {
            ans[i++] = num;
        }

        return ans;
    }
}

File: Data Structures & Algorithms/palindromic-substrings/submission-0.py
class Solution:
    def countSubstrings(self, s: str) -> int:
        count=0
        for i in range(len(s)):
            r=i
            l=i
            while(l>=0 and r<len(s) and s[l]==s[r]):
                count+=1
                l-=1
                r+=1
            r=i+1
            l=i
            while(l>=0 and r<len(s) and s[l]==s[r]):
                count+=1
                l-=1
                r+=1
        return count
            
            